index.html

```diff
@@ -1,19 +1,147 @@
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Store</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Blog</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Help</a>
+            </li>
+            <li class="list-item">
+              <a href="./order.html" class="nav-link">Login</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="home">
+        <div class="content">
+          <h2 class="intro-karma">Introducing Karma</h2>
+          <p class="slogan">Bring WiFi with you everywhere you go.</p>
+          <button class="learn-more">Learn More</button>
+        </div>
+      </section>
+      <section class="products">
+        <h2>Everyone needs a little Karma.</h2>
+        <div class="product-cards">
+          <article class="cards">
+            <img
+              class="card-logos"
+              src="./img/icon-devices.svg"
+              alt="devices-logo"
+            />
+            <p>Internet for all devices</p>
+          </article>
+          <article class="cards">
+            <img
+              class="card-logos"
+              src="./img/icon-coffee.svg"
+              alt="coffee-logo"
+            />
+            <p>Boost your productivity</p>
+          </article>
+          <article class="cards">
+            <img
+              class="card-logos"
+              src="./img/icon-refill.svg"
+              alt="refill-logo"
+            />
+            <p>Pay as you go</p>
+          </article>
+        </div>
+      </section>
+      <section class="quoto-section">
+        <img
+          class="quoto-image"
+          src="./img/homepage-feature.jpg"
+          alt="quoto image"
+        />
+        <article class="quoto-container">
+          <div class="quoto-content">
+            <p>
+              <span>“</span>Wherever I am, I just don’t worry about my
+              connection anymore!<span>”</span>
+            </p>
+            <button class="get-karma">Get Karma today</button>
+          </div>
+        </article>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-container">
+        <h3>Join us on</h3>
+        <div class="contact-icons">
+          <a href="#">
+            <img
+              class="footer-icons"
+              src="./img/twitter-icon.svg"
+              alt="twitter-icon"
+            />
+          </a>
+          <a href="#">
+            <img
+              class="footer-icons"
+              src="./img/facebook-icon.svg"
+              alt="facebook-icon"
+            />
+          </a>
+          <a href="#">
+            <img
+              class="footer-icons"
+              src="./img/instagram-icon.svg"
+              alt="instagram-icon"
+            />
+          </a>
+        </div>
+        <p>@Karma mobility, INC</p>
```

Comment:
You can replace the @ sign with `&copy;` and it will render as a copyright symbole